src/components/MaterialUI/Inputs/DatePicker.js

```diff
@@ -79,7 +85,7 @@ const WrappedDatePicker = ({
 	};
 
 	return (
-		<label className={classes.datePickerWrapper}>
+		<label className={clsx(classes.datePickerWrapper, disabledCls)}>
```

Comment:
@Crispried In some places, we use "classNames" and some other, we use "clsx". Do we have any standard for that?

Reply:
@joclefrancois @Crispried  In this case clsx is used to display conditional/complex classes. For example if disabledCls is empty it will not be added to the className property.

Reply:
after review in orc-shared, will be using classNames